.github/workflows/deploy-virus-scanner-guardduty-prod.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy Virus Scanner GuardDuty to IaC OGP AWS environment
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - release-al2
+      - test/migrate-prod-to-iac
```

Comment:
To remove this before merging, used currently before review approval to test and trigger IaC prod deployments.